runtime/service_integ_test.go

```diff
@@ -265,7 +263,9 @@ func TestMultipleVMs_Isolated(t *testing.T) {
 	// container ends up in the right VM by assigning each VM a network device with a unique mac address and having each container
 	// print the mac address it sees inside its VM.
 	var vmWg sync.WaitGroup
-	for vmID, c := range cases {
+	for i := 0; i < numberOfVms; i++ {
+		caseTypeNumber := 1 % 2
+		c := cases[caseTypeNumber]
 		vmWg.Add(1)
 		go func(vmID int, containerCount int32, jailerConfig *proto.JailerConfig) {
```

Comment:
I am also curious to how many are we actually running in parallel due to the `StopVM` call. We may need to call `StopVM` at a later time and have the VMs run some process to ensure we are getting accurate number of concurrency. Thoughts on this @sipsma

Reply:
I don't think this is meant to be a true scale test, so we don't really need to max out the host capacity. AFAICT, 100 was chosen pretty much arbitrarily, and it is an upper bound on the number of VMs that will run concurrently, not necessarily a goal. Even if we're only running ~20 VMs at any one time, that's still more than we're running today. If we assume that more VMs means more likelihood of catching concurrency-related bugs, then a 3-4x increase in the number of running VMs sounds good to me.

At some point in the future we will need to be performing actual scale tests, but this PR doesn't need to be that.

Reply:
> I'm worried that a lot of the VMs are exiting before we can test the true bottleneck of what is parallel

This very well may be the case, but I agree with Noah that it's fine if so. We are primarily looking to find problems that occur as we actually perform actions on/in the VM (spin up/down containers, copy io, etc.). So even if we modified the test case to wait to call StopVM at the end, we would likely just end up with a bunch of VMs sitting around doing nothing for a while, which doesn't really improve coverage IMO. I agree though that if we want scale tests in the future we'd need to be more careful about issues like this.

Reply:
Ah, I was under the impression we wanted to actually test load. In that case, this looks fine